beacon/sync/src/main/java/tech/pegasys/teku/beacon/sync/forward/singlepeer/PeerSync.java

```diff
@@ -171,7 +171,8 @@ private SafeFuture<PeerSyncResult> executeSync(
               final PeerSyncBlobSidecarListener blobSidecarListener =
                   new PeerSyncBlobSidecarListener(requestContext.startSlot, requestContext.endSlot);
 
-              if (blobSidecarManager.isAvailabilityRequiredAtSlot(requestContext.endSlot)) {
+              if (blobSidecarManager.isStorageOfBlobSidecarsRequiredAtSlot(
```

Comment:
This has the same problem of the historical sync, but I think is more critical here: we could stall in forward sync if we are way outside the standard DA and very few\no peers are serving this blobs.